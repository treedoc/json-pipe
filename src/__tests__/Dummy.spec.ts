describe('Dummy test', () => {
  test('Just to satisfy compile', () => {
    const s = 'hello';
    expect(s).toMatchSnapshot();
  });
});
