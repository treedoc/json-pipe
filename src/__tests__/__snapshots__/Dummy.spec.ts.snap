// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dummy test Just to satisfy compile 1`] = `"hello"`;
